tests/trig.rs

```diff
@@ -415,6 +620,11 @@ mod test {
         let complex: Ruleset<Trig> = Ruleset::from_file("scripts/trig/complex.rules");
         assert_eq!(complex.len(), 57);
 
+        let limits = Limits {
+            iter: 3,
```

Comment:
do we still use adhoc values here? should we set them to something fixed somewhere for all domains?

Reply:
These are adhoc. Much higher than default.